### ISSUE TITLE
Dangerous "kill me now!" button in Sunstone Actions tab

### ISSUE THREAD

**Issue by Yenya** [OpenNebula/one#5294]
**Description**
There is a very dangerous button in Sunstone in the VM page (Instances->VMs->select a particular VM), the Actions tab.
The button with blue clock icon, when clicked, adds two new actions to the current VM:

* suspend after two weeks
* delete after four weeks

The two and four weeks, respectively, are counted from the time when that VM has been created. So it means that if one has a VM which has been running for more than four weeks, this button destroys the VM and its data immediately, no questions asked.
This is a pretty serious UI bug, I think.

**To Reproduce**
Have a VM with more than 4 weeks of uptime.
Open its Actions tab, click on the clock icon.
Pull hair wondering what the configuration of that VM was, where is the backup of its disks, and how did it disappear so fast.

**Expected behavior**
The button should be black, it should have a meaningful tooltip ("Kill that VM now!"), and the icon should be a big skull or something :-)

Seriously, the button should be removed altogether, or at least set up the actions to be two weeks from the current time, not from the VM startup time. It should also have a tooltip and maybe a confirmation dialog (the same way as Terminate button has).

**Details**
 - Affected Component: Sunstone
 - Hypervisor: KVM
 - Version: 5.12.0.3 CE
 - 
**Additional context**

## Progress Status
- [ ] Branch created 
- [ ] Code committed to development branch
- [ ] Testing - QA
- [ ] Documentation 
- [ ] Release notes - resolved issues, compatibility, known issues
- [ ] Code committed to upstream release/hotfix branches
- [ ] Documentation committed to upstream release/hotfix branches

**Comment by Yenya** [comment]
This is also related to #5024.

**Comment by atodorov-storpool** [comment]
duplicate of #5209
@Yenya you could disable this "feature" commenting out the "leases" in `/etc/one/sunstone-server.conf`
```
# this display button and clock icon in table of vm
#:leases:
#  suspense:
#    time: '+1209600'
#    color: '#000000'
#    warning:
#      time: '-86400'
#      color: '#085aef'
#  terminate:
#    time: '+1209600'
#    color: '#e1ef08'
#    warning:
#      time: '-86400'
#      color: '#ef2808'
```

**Comment by Yenya** [comment]
Anton, thanks for the notice. I have missed #5209 indeed, sorry.

Commenting out the :leases: part in `sunstone-server.conf` works (the word `undefined` is displayed instead of the clock icon, though).

**Comment by tinova** [comment]
The blue clock refer to the VM Charter functionality:
  
  http://docs.opennebula.io/5.12/operation/vm_management/vm_instances.html#vm-charter

This suffers from a nasty bug as you have experienced, this is fixed already and will be ready in future releases (#5209). 

We'll take a look at the "undefined" text display when disabled.

**Comment by jloboescalona2** [comment]
@tinova PR to approve:
 - Code: https://github.com/OpenNebula/one-ee/pull/948
 - Docs: https://github.com/OpenNebula/docs/pull/1509

**Comment by tinova** [comment]
fixed in master and hence 6.0
